File: python/ex_basic/Session_one/demo3.py
import math
print("Input X1: ", end=" ")
x1 = int(input())
print("Input y1: ", end=" ")
y1 = int(input())
print("Input x2: ", end=" ")
x2 = int(input())
print("Input y2: ", end=" ")
y2 = int(input())

oxy = '%.2f' % (math.sqrt(pow(x2-x1, 2)+pow(y2-y1, 2)))
print("Khoang Cach = {0}".format(oxy))

File: class/java/home/ex_home/session_one/demo3.java
import java.util.Scanner;
import java.lang.Math;
import java.text.DecimalFormat;

public class demo3 {
    public static void main(String[] agrs){
        Scanner scanner = new Scanner(System.in);
        DecimalFormat decimalFormat = new DecimalFormat("0.00");

        System.out.println("Nhap x1: ");
        int x1 = scanner.nextInt();
        System.out.println("Nhap x2: ");
        int x2 = scanner.nextInt();
        System.out.println("Nhap y1: ");
        int y1 = scanner.nextInt();
        System.out.println("Nhap y2: ");
        int y2 = scanner.nextInt();

        float cal = (float)Math.sqrt(Math.pow((x2-x1*1.0), 2)+Math.pow((y2-y1*1.0), 2));
        System.out.println("Khoang cach giua hai diem la: "+ decimalFormat.format(cal) );
    }
}
